docs/wmco-prerequisites.md

```diff
@@ -18,11 +18,14 @@ applicable cloud provider.
 Note: Any unlisted Windows Server version are NOT supported, and will cause errors. To prevent 
 these errors, only use the appropriate version according to the cloud provider in use. 
 
-| Cloud Provider | Supported Windows Server version                                       |
-|----------------|------------------------------------------------------------------------|
-| AWS            | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019 |
-| Azure          | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019 |
-| VMware vSphere | Windows Server Semi-Annual Channel (SAC): Windows Server 20H2          |
+| Cloud Provider      | Supported Windows Server version                                                     |
+| -----------         | -----------                                                                          |
+| AWS                 | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019, version 1809 |
+| Azure               | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019, version 1809 |
+| VMware vSphere      | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2022               |
+|                     | Windows Server Semi-Annual Channel (SAC): Windows Server 20H2                        |
```

Comment:
When rendered, this line appears in a new row which could be confusing.

Reply:
when do we remove 20H2 from the list? After validating  2022 in AWS and AZURE ?

Reply:
When it goes out of support from MSFT’s end, 2022-08-09